src/dev-core/src/app/button/button.demo.ts

```diff
@@ -4,10 +4,33 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component } from '@angular/core';
+import { ClrLoadingState } from '@clr/core';
 import '@clr/core/button';
+import '@clr/core/icon';
```

Comment:
Small non blocking nit but I don't think you have to import the icon again since it is imported in the button demo module.

Reply:
Thanks for catching that. Removing.